addon/commands/insertAndCollapse.js

```diff
@@ -0,0 +1,32 @@
+export default class InsertAndCollapseCommand {
+  name = 'insert-and-collapse';
+
+  constructor(model) {
+    this.model = model;
+  }
+
+  canExecute() {
+    return true;
+  }
+
+  execute(controller, htmlString, node) {
+    const range = controller.rangeFactory.fromInNode(
+      node,
+      0,
+      node.getMaxOffset()
+    );
+    controller.executeCommand('insert-html', htmlString, range);
```

Comment:
were commands supposed to call other commands? do we have other instances where we do this?

Reply:
This was a discussion with Arne at some point, we decided that they could I think, but we can still discuss it